src/pudl/workspace/datastore.py

```diff
@@ -435,12 +447,13 @@ def _get_pudl_in(args: dict) -> Path:
 def _create_datastore(args: dict) -> Datastore:
     """Constructs datastore instance."""
     local_cache_path = None
-    if not args.populate_gcs_cache:
+    if not args.populate_gcs_cache and not args.populate_s3_cache:
```

Comment:
Either or is fine, just suggesting an alternative
```suggestion
    if not (args.populate_gcs_cache or args.populate_s3_cache):
```